dashboard/app/models/user.rb

```diff
@@ -109,6 +109,8 @@ class User < ActiveRecord::Base
     races
     using_text_mode
     last_seen_school_info_interstitial
+    ui_tip_dismissed_homepage_header
+    ui_tip_dismissed_teacher_courses
```

Comment:
Do we have a plan to sunset these serialized properties? Presumably, at some point we want to stop showing those tooltips and remove this serialized attribute? Assuming this is the case, do we know a rough timeline as to when this should happen?